src/site/_filters/paged-navigation.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Takes an object and determines how many links to other pages in the same paginated collection to display.
+ * @param {object} paged Page details including the `index` in the pagination, how many `pages` there are, and the `href` of the page.
```

Comment:
Use better JSDoc for this. One option:

```
/**
  * @param {object} paged Page details
  * @param {number} paged.index Index of ...
  */
```

Or you can do the wholly-inline approach e.g `{{index: number, pages: number, href: string}}`. Take [a look here for options](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#typedef-callback-and-param).

Reply:
is this done?